hack/run-functional-tests.sh

```diff
@@ -172,7 +212,10 @@ run_resource_tests() {
 
         else
 
-            echo "[WARN] File \"${test_file_path}\" not found. Skipping..."
+            # try to warn that the functional test manifest has a problem with one of the file definitions
+            echo "[WARN] File \"${testfile}\" not found in directory \""testing/${resource}/"\". Functional Testing Failed..."
```

Comment:
Looks like there's some extra double quotes in this that's causing shellcheck to fail

```suggestion
            echo "[WARN] File \"${testfile}\" not found in directory \"testing/${resource}/\". Functional Testing Failed..."
```

Reply:
Fixed in more recent commit